msm/src/main.rs

```diff
@@ -10,7 +10,7 @@ use kimchi_msm::precomputed_srs::get_bn254_srs;
 use kimchi_msm::prover::prove;
 use kimchi_msm::verifier::verify;
 use kimchi_msm::{
-    BN254G1Affine, BaseSponge, Ff1, Fp, OpeningProof, ScalarSponge, BN254, DOMAIN_SIZE,
+    BN254G1Affine, BaseSponge, Ff1, Fp, OpeningProof, ScalarSponge, BN254, DOMAIN_SIZE, N,
```

Comment:
I would probably call this `N` something different here. For example, in kimchi we had something like `COLUMNS` or `KIMCHI_COLS` so it is more meaningful. Maybe `N` can be the name for the generic constant in the traits/functions, but the concrete value of it for MSM can be called `MSM_COLS` or similar.

Reply:
Follow-up https://github.com/o1-labs/proof-systems/issues/1881